[docs] Getting started review (#165)
cozystack/website: content/en/docs/getting-started/first-deployment.md

## Objectives

This tutorial shows how to bootstrap Cozystack.
It will guide you through the following steps:

> 1.  Create Kubernetes cluster

Reviewer: ```suggestion
1.  Create a Kubernetes cluster
```

1.  Bootstrap Cozystack
1.  Configure Storage
1.  Configure Networking
1.  Access Cozystack dashboard
1.  Deploy etcd, ingress and monitoring stack

## Create Kubernetes cluster

Cozystack runs on top of a Kubernetes cluster, either bring one up with Talos, or use an existing one.

### Install Talos Linux

Boot your machines with Talos Linux image in one of these ways:

> - [Install using temporary DHCP and PXE servers](/docs/operations/talos/installation/pxe/) running in a Docker container.

Reviewer: There are two servers, as far as I understand.

```suggestion
- [Install using temporary DHCP and PXE servers](/docs/operations/talos/installation/pxe/) running in Docker containers.
```

- [Install using ISO](/docs/operations/talos/installation/iso/).
- [Install on Hetzner servers](/docs/operations/talos/installation/hetzner/).

### Bootstrap Talos cluster

Bootstrap your Talos Linux cluster using one of the following tools:

- [**talos-bootstrap**](/docs/operations/talos/configuration/talos-bootstrap/), for an interactive walkthrough.
- [**Talm**](/docs/operations/talos/configuration/talm/), for a declarative way of cluster management.

> 

Reviewer: Is https://github.com/cozystack/talos-bootstrap/ really interactive?

### Existing cluster or other Kubernetes distributions

If you bootstrap your Talos cluster in your own way, or use a different Kubernetes distribution, make sure
to apply all settings from the guides above.

> These are a mandatory:

Reviewer: ```suggestion
These are mandatory:
```
